tvappweb/components/AppHead/AppHead.js

```diff
@@ -68,6 +68,11 @@ function AppHead() {
     d.getElementsByTagName('head')[0].appendChild(s);
   }, []);
 
+  useEffect(() => {
+    // Feeding Chatbot with an initial event when website page loads.
+    window.$crisp.push(['set', 'session:event', [[['chaseoakstv:page:loaded', {}, 'red']]]]);
```

Comment:
```suggestion
    window.$crisp.push([
      'set',
      'session:event',
      [[['chaseoakstv:page:loaded', {}, 'red']]],
    ]);
```

@leungwaynetx FYI, our linter is wanting it to be formatted like this - it's supposed to automatically add these suggestions to the PR but for some reason that's not working, so I've added the code suggestion manually in this comment.

You should be able to accept the suggestion above and the PR checks will re-run and then we should be good to go.